pkg/controller/manager/manager_controller.go

```diff
@@ -271,19 +272,23 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 		}
 	}
 
-	// Check that compliance is running.
-	compliance, err := compliance.GetCompliance(ctx, r.client)
-	if err != nil {
-		if errors.IsNotFound(err) {
-			r.status.SetDegraded("Compliance not found", err.Error())
+	var installCompliance = utils.IsFeatureActive(license, common.ComplianceFeature)
```

Comment:
What happens when my license doesn't have this capability, but then I later apply a new license that has the feature enabled? How will we detect this and update the installed components?

Are we just relying on the compliance_controller's 10s retry timer to detect and update the status, thus triggering the manager as well?

I suppose that will probably be good enough, in the absence of a true watch for the License itself.

Reply:
@caseydavenport Yes, it will install the component if a license is being installed that has that feature.